test/tests/functional/pbs_alps_release_tunables.py

```diff
@@ -92,16 +93,14 @@ def test_alps_release_wait_time(self):
         # to either run it again or find out a way to delay the reservation
         # cancellation at ALPS level itself.
         if len(out) >= 2:
-            _, line_prev = out[0]
-            time_prev = get_epoch(line_prev)
+            time_prev = get_epoch(out[0][1])
             for data in out[1:]:
-                _, line_current = data
+                line_current = data[1]
```

Comment:
You only use line_current once, just use data[1] there instead.

Reply:
I don't see how it is more readable.  The only thing it provides is the name line_current.  If you want more explanation, use a comment.

Reply:
Well, you are right, a comment would have helped but I feel it is not complex enough that it would need a comment and it is also not clear enough that data[1] is the line where log_match() finds a hit. I thought the variable name does help here. 
I'd like to know if @lisa-altair and @vccardenas also think it should be removed then I'll remove it.
I personally like to be lenient with variable use in PTL script because I feel it does not save us much to have it optimized and everyone has their own personal style of dealing with things, but I may be wrong :)